### DISCUSSION TITLE
Rename files-Question

### DISCUSSION THREAD

**Discussion by Tvirin** [hardcore-sushi/DroidFS#126] [Q&A]
When the name of the encrypted file is changed from file manager, it is no longer displayed in the application. This can change someone and I lose access to the files.

Is there a way to reopen the file with the changed name?

**Comment by hardcore-sushi** [comment]
I think you should better ask your question to the developer of gocryptfs: https://github.com/rfjakob/gocryptfs

**Reply by Tvirin** [reply]
Thanks sir, I'll ask. 

_I hope you understood what I mean._

**Reply by hardcore-sushi** [reply]
Yes I got it, but this is more of a gocryptfs related question than a DroidFS related question.
